SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs

```diff
@@ -12,7 +15,15 @@ public class GameController : Controller
     {
         private ApplicationDbContext db = new ApplicationDbContext();
 
+        //Dictionary containing ESRB ratings
+        public static List<string> esrbList = new List<string>
+            {
+                {"EC"},{"E"},{"E10"},{"T"},{"M"},{"AO"}
```

Comment:
I originally had a dictionary then decided a list would be sufficient. Can't fix the comment atm, I dislike using var and avoid it whenever possible.